packages/nimble-components/src/rich-text-viewer/template.ts

```diff
@@ -2,5 +2,7 @@ import { html } from '@microsoft/fast-element';
 import type { RichTextViewer } from '.';
 
 export const template = html<RichTextViewer>`
-    <template> Rich Text Viewer here. </template>
+    <template>
+        <div class="container" id="viewer"></div>
```

Comment:
Normally we set `id` in our template code because there is a functional dependency on it being set either for an anchored region or a label. If its reason for existence is purely test-driven, it's usually fine to just use the `class` value if there is one available.

Reply:
Removed `id` and by using the `class` to update the view for the component.